deno_webgpu/src/lib.rs

```diff
@@ -242,13 +242,14 @@ pub async fn op_webgpu_request_adapter(
 ) -> Result<GpuAdapterDeviceOrErr, AnyError> {
     let mut state = state.borrow_mut();
     check_unstable(&state, "navigator.gpu.requestAdapter");
+    let backends = std::env::var("WGPU_BACKEND").ok().and_then(|s| wgpu_types::Backends::from_comma_list(&s)).unwrap_or_else(wgpu_types::Backends::all);
```

Comment:
```suggestion
    let backends = std::env::var("DENO_WEBGPU_BACKEND").ok().and_then(|s| wgpu_types::Backends::from_comma_list(&s)).unwrap_or_else(wgpu_types::Backends::all);
```

All the environment variables Deno cares about should be prefixed by `DENO_`